web/Dockerfile

```diff
@@ -37,6 +37,7 @@ FROM gcr.io/linkerd-io/base:2017-10-30.01
 COPY --from=golang /go/src/github.com/linkerd/linkerd2/web .
 RUN mkdir -p ./dist
 COPY --from=webpack-bundle /go/src/github.com/linkerd/linkerd2/web/app/dist ./dist
+COPY LICENSE /linkerd/LICENSE
```

Comment:
Since we don't expect the license file to change, can you move this line immediately after the `FROM` on line 36? That will allow it to be cached between builds. And can you make similar changes in the other Dockerfiles as well?